### PR TITLE
Use classSet to concat class names.

### PR DESCRIPTION
**Pull request by Gozala**

### CODE REVIEW

File: src/browser/navigation-panel.js [hunk 1]
```diff
@@ -100,12 +101,15 @@ define((require, exports, module) => {
...
-                            (webViewerCursor.get('isLoading') ? ' loading' : '') +
-                            (webViewerCursor.get('securityState') == 'secure' ? ' ssl' : '') +
-                            (webViewerCursor.get('securityExtendedValidation') ? ' sslv' : '')
+      className: ClassSet({
+        navbar: true,
+        urledit: inputCursor.get('isFocused'),
+        cangoback: webViewerCursor.get('canGoBack'),
+        canreload: webViewerCursor.get('location'),
+        loading: webViewerCursor.get('isLoading'),
+        ssl: webViewerCursor.get('securityState'),
```
Comment (gordonbrander) [inline src/browser/navigation-panel.js]: We used to compare `webViewerCursor.get('securityState') == `secure`. What is the contents of `webViewerCursor.get('securityState')`? `secure` or `null`? Want to make sure we're not introducing a state bug.

Comment (Gozala) [inline src/browser/navigation-panel.js]: Good catch, I'll fix that

**Comment by gordonbrander** [comment]
Looks good. One question I have is how often we're creating these class description objects? Once per render? I'm not too worried about the garbage -- after all, garbage collector is there to pick up garbage -- but it's nice to understand what is generating garbage and how much.

**Comment by Gozala** [comment]
> Looks good. One question I have is how often we're creating these class description objects? Once per render? I'm not too worried about the garbage -- after all, garbage collector is there to pick up garbage -- but it's nice to understand what is generating garbage and how often.

Yes it's per render, which should be triggered only if state we're rendering has changed.

**Pull request merged**